app/models.py

```diff
@@ -0,0 +1,32 @@
+from sqlalchemy import (
+    Column,
+    DateTime,
+    Float,
+    ForeignKey,
+    Integer,
+    String,
+)
+from sqlalchemy.orm import relationship
+
+from app.core.database import Base
+
+
+class DBCity(Base):
```

Comment:
[changed request] In FastAPI, we use models.City and schemas.City. Therefore, you should rename it to City.